### PR TITLE
chore: add news for the release 2.5

### PR DESCRIPTION
**Pull request by e0**
**Related issue(s) and PR(s)**  
This PR  relates to #941 .

**Type of change**  

- [x] Other
 
**List of changes made**  

**Screenshot of the fix**  

**Testing**  

**Further comments**  

**Definition of Done checklist**  
- [x] My code meets the Acceptance Criteria
- [x] My code follows the [NBIS style guidelines](https://github.com/NBISweden/development-guidelines)
- [x] I have performed a self-review of my own code and commented any hard-to-understand areas
- [ ] Tests and lint/format validations are passing
- [x] My changes generate no new warnings

### CODE REVIEW

File: frontend/src/content/news.json [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+      "text": "<b>Metabolic Atlas 2.4</b> is out. In this minor release we have focused on improving the search functionality. The improvement has been achieved for both the <i>GEM Search</i> and the <i>Global Search</i> by a better scoring of searching items and a more logical presentation of found results. Moreover, we have resolved a number of small bugs that affected the user experience. More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/2.4' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/record/6378851#.Ykr4Fy8RpB0' target='_blank'>on Zenodo</a>."
```
Comment (mihai-sysbio) [inline frontend/src/content/news.json:11]: Looking at the previous links, I don't think the anchor is needed.

```suggestion
      "text": "<b>Metabolic Atlas 2.4</b> is out. In this minor release we have focused on improving the search functionality. The improvement has been achieved for both the <i>GEM Search</i> and the <i>Global Search</i> by a better scoring of searching items and a more logical presentation of found results. Moreover, we have resolved a number of small bugs that affected the user experience. More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/2.4' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/record/6378851' target='_blank'>on Zenodo</a>."
```

File: frontend/src/content/news.json [hunk 2]
```diff
@@ -1,9 +1,14 @@
 {
   "2022": [
+    {
+      "date": "2022-04-04",
+      "title": "Metabolic Atlas 2.5 improves the about pages and fixes bugs",
```
Comment (mihai-sysbio) [inline frontend/src/content/news.json:5]: In news text we mention `<i>About</i> pages`capitalized and in italics, so it is unexpected to called them here `about pages`.

File: frontend/src/content/news.json [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+      "text": "<b>Metabolic Atlas 2.5</b> is out. In this minor release we have focused on maintenance, fixing a number of bugs related to search and routing. General improvements for the content pages, including the <i>About</i> pages and <i>Documentation</i>, have been added as well. More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/2.5' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```
Comment (mihai-sysbio) [inline frontend/src/content/news.json:6]: `routing` sounds too dev oriented for the news, are there any other ideas?

Comment (nanjiangshu) [reply frontend/src/content/news.json:6]: `routing` is the reasoning of the bug and in the news I think it is better to describe the bug itself. What about `broken links` or `broken URLs`?

**Review by nanjiangshu — COMMENTED** [review]
Looks great @e0! Apart from some minor comments from me and other reviewers I think it is ready to be merged.

**Review by MalinAhlberg — APPROVED** [review]
Looks great (after fixing the two minor suggestions)!

File: frontend/src/content/news.json (hunk 1 above)
Comment (e0) [reply frontend/src/content/news.json:11]: Ah that makes sense. Thanks! I was wondering why that part was constantly changing.

File: frontend/src/content/news.json (hunk 3 above)
Comment (e0) [reply frontend/src/content/news.json]: `broken links` should be great, thanks! Addressed in: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/942/commits/4f9a85ada008534320266fc9ac61b4eb90da681a

File: frontend/src/content/news.json (hunk 2 above)
Comment (e0) [reply frontend/src/content/news.json]: Thanks and sorry that I missed this before re-requesting reviews. Changed to `About` in https://github.com/MetabolicAtlas/MetabolicAtlas/pull/942/commits/b4d86757289113671bd36523fdd3dfa43245e83c now. The title currently doesn't parse any html tags. Do you want us to allow that so we can make it italic as well?

Comment (mihai-sysbio) [reply frontend/src/content/news.json]: For consistency, I think it would make sense, not sure if we need a separate issue.

Comment (e0) [reply frontend/src/content/news.json]: Thanks for the quick response! I felt like it was relevant and small enough to be included so I added it here: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/942/commits/75cab8f594b17aafdc2a238b2096870004f9e364

Comment (mihai-sysbio) [reply frontend/src/content/news.json]: Awesome!

**Review by mihai-sysbio — APPROVED** [review]
🌟

**Pull request merged**